src/readii/feature_extraction.py

```
@@ -1,392 +1,446 @@
+	Returns
+	-------
+	pd.DataFrame
+		Dataframe containing the image metadata and extracted radiomic features.
+	"""
+
+	# Setting pyradiomics verbosity lower
+	radiomics_logger: logging.Logger = logging.getLogger("radiomics")
+	radiomics_logger.setLevel(logging.ERROR)
+
+	# If no pyradiomics paramater file passed, use default
+	if pyradiomicsParamFilePath == None:
```

Ruff (0.8.0)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)